### PR TITLE
Fix spring boot test using wrong TestConfiguration class for loading ApplicationContext

### PR DESCRIPTION
**Pull request by kojilin**
Found maybe using unexpected class for loading ApplicationContext when checking flaky test. (This doesn't fix the flaky test I think, just make the code using correct class)

* Change package name of Configuration class importing
* Remove durplicate `OkService`

### CODE REVIEW

File: spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSslConfigurationTest.java [hunk 1]
```diff
@@ -64,15 +61,6 @@
 
     @SpringBootApplication
     public static class TestConfiguration {
-
-        @Bean
-        public ArmeriaServerConfigurator okService() {
```
Comment (kojilin) [inline spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSslConfigurationTest.java:69]: Already has a ArmeriaOkServiceConfiguration in same package.

File: spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSslConfigurationTest.java [hunk 2]
```diff
@@ -64,15 +61,6 @@
 
     @SpringBootApplication
     public static class TestConfiguration {
```
Comment (minwoox) [inline spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSslConfigurationTest.java:63]: nit: can remove `public` here as well?

File: spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSslConfigurationTest.java (hunk 1 above)
Comment (minwoox) [reply spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSslConfigurationTest.java:69]: Then could you remove the `OkService` underneath, please?

**Review by minwoox — APPROVED** [review]
What a bunch of mistakes! 😅 
Thanks a lot!

**Review by ikhoon — APPROVED** [review]
Thanks @kojilin !

**Pull request merged**